src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -679,6 +778,27 @@ public void serializeViewNetworkData(@NotNull final ByteBuf buf)
         buf.writeDouble(getSaturation());
 
         ByteBufUtils.writeUTF8String(buf, (job != null) ? job.getName() : "");
+
+        final EntityCitizen citizen = getCitizenEntity();
+        if (citizen != null)
+        {
+            final ITextComponent[] latestStatus = citizen.getLatestStatus();
+            buf.writeInt(latestStatus.length);
+            for (int i = 0; i < latestStatus.length; i++)
+            {
+                ByteBufUtils.writeUTF8String(buf, latestStatus[i] == null ? "" : latestStatus[i].getUnformattedText());
+            }
+        }
+        else
+        {
+            buf.writeInt(0);
+        }
+
+        buf.writeInt(colony.getID());
+
+        NBTTagCompound compound = new NBTTagCompound();
+        compound.setTag("inventory", inventory.writeToNBT(new NBTTagList()));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'TAG_INVENTORY' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1192)